web/src/routes/SurveyPages/SurveyListPage.tsx

```diff
@@ -1,5 +1,201 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
+import { AxiosResponse } from 'axios';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+
+import axios from '../../api/axios';
+import requests from '../../api/request';
+import Header from '../../components/Header';
+import { useTheme } from '../../hooks/useTheme';
+
+const Container = styled.div`
+  width: 100vw;
+  height: 100vh;
+  background-color: ${(props) => props.theme.colors.container};
+`;
+
+const ListContainer = styled.div`
+  width: 90vw;
+  height: 94vh;
+  display: flex;
+  flex-direction: column;
+  padding: 3vh 5vw 3vh 5vw;
+  background-color: ${(props) => props.theme.colors.container};
+`;
+
+const ListColumn = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+const Item = styled.div`
```

Comment:
This should be `td`. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table

And for other tags, please change or add proper tags!